src/main/test/java/com/salesforce/E2ETest.java

```diff
@@ -0,0 +1,27 @@
+package com.salesforce;
+
+import com.github.charithe.kafka.EphemeralKafkaCluster;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class E2ETest {
+    @Test
+    public void e2eTest() throws Exception {
+        EphemeralKafkaCluster cluster = EphemeralKafkaCluster.create(1);
+        long maxWaitMs = TimeUnit.MINUTES.toMillis(1);
+        for (long wait = 0; wait < maxWaitMs; wait += TimeUnit.SECONDS.toMillis(1)) {
+            if (cluster.isRunning() && cluster.isHealthy()) {
+                break;
+            } else {
+                TimeUnit.SECONDS.sleep(1);
+            }
+        }
+        while (true) {
+            System.out.println(cluster.producerConfig());
```

Comment:
We can actually run your new `BenchmarkApp` here and do a quick sanity check

Reply:
I couldn't do this, because the library assigns a random port to Kafka. I can spend more time later once I complete this task to scan the port etc. But if you know an easy way, let me know.

Reply:
Ahhh, right. We'd have to make the config more dynamic, so more refactoring. We can do that in a future PR then.

Reply:
Yeah. Just want to minimize the changes in this one. I can file a Jira.